src/todos.js

```diff
@@ -10,12 +10,12 @@ const Todos = ({todos, deleteTodo}) => {
             )
         })
     ) : (
-        <p className="center">You have no todo's left!</p>
+        <p className="center">You have no todoo's lefy!</p>
```

Comment:
There are a couple of typos in the text. It should be 'You have no todos left!', not 'You have no todoo's lefy!'.